### PR TITLE
Fix RSS links

### PR DESCRIPTION
**Pull request by ostrolucky**
Hello, RSS links don't work at all, this should fix this.

### CODE REVIEW

File: src/RssBuilder.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public function build(array $emails) : string
         foreach ($emails as $email) {
             $item = $this->dom->createElement('item');
             $this->addTextNode('title', $email->getSubject(), $item);
-            $this->addTextNode('link', $this->host . $email->getUrl(), $item);
+            $this->addTextNode('link', $this->host . '/message/' . $item['number'], $item);
```
Comment (mnapoli) [inline src/RssBuilder.php]: `$item` is a DomNode, I don't see why you do `['number']` on it 🤔 that shouldn't really work

Maybe it should be `$email->getNumber()` instead?

(ping @agallou who may have a better idea for this code)

Comment (mnapoli) [inline src/RssBuilder.php]: `$item` is a DomNode, I don't see why you do `['number']` on it 🤔 that shouldn't really work

Maybe it should be `$email->getNumber()` instead?

(ping @agallou who may have a better idea for this code)

**Comment by staabm** [comment]
just a note: since the application wasn't deployed for a few weeks it might also be a bug showing up, because the latest source code is not on LIVE yet.

File: src/RssBuilder.php (hunk 1 above)
Comment (ostrolucky) [reply src/RssBuilder.php]: Correct, I guess it was late at night. I have changed it

**Pull request merged**